app/main/routes.py

```diff
@@ -64,6 +64,19 @@ def faq():
 
     return render_template('faq.html', title='CONP | FAQ', user=current_user, content=content)
 
+@main_bp.route('/result')
+def result():
```

Comment:
would it be possible to give a name that is more information for the result function and page?

Example: cbrain_analytics or something like that?

Reply:
Hi Cecile, the point is that these results are not for CBRAIN, these the provenance records generated from running the pipelines in boutiques. Yeah I will change the name to boutiques_analytics

Reply:
I see. Then boutiques_analytics sounds like a good name. Thanks!

Reply:
"Pipeline execution analytics" sounds even better.